.github/actions/build-dotnet/action.yml

```diff
@@ -97,6 +105,15 @@ runs:
         key: ${{ steps.detect-cache-configuration.outputs.key }}
         restore-keys: ${{ steps.detect-cache-configuration.outputs.restore-keys }}
 
+    - name: PowerShell Static Code Analysis
```

Comment:
We need to run this for PRs opened in this repo too.

Reply:
It is ran on OSOCE and this repo is a submodule of it, so it's ran that way. Or do you mean something else?

Reply:
It feels a bit strange not to have a standalone build for this repo and depend on using it as a submodule (because you can work on it without OSOCE, as opposed to the OC projects) but I see the point.